api/grpcserver_test.go

```diff
@@ -2379,10 +2379,13 @@ func TestGrpcServer_GetTransactionLogByActionHash(t *testing.T) {
 		require.Equal(log.Proto(), res.TransactionLog)
 	}
 
-	// check implicit transfer receiver balance
-	state, err := accountutil.LoadAccount(svr.core.StateFactory(), identityset.Address(31))
-	require.NoError(err)
-	require.Equal(big.NewInt(5), state.Balance)
+	// TODO (huof6829): Re-enable this test
```

Comment:
Is this test failed?

Reply:
The purpose of this PR is to delete `StateFactory` interface.

Reply:
since the PR is to delete SF interface, i think can just delete this code block, NO need to leave a TODO. We must have tests (in other places) for implicit transfer